rpgJavaInterpreter-core/src/test/resources/mute/MUTE13_41.rpgle

```diff
@@ -0,0 +1,29 @@
+     V*=====================================================================
+     V* MODIFICHE Ril.  T Au Descrizione
+     V* gg/mm/aa  nn.mm i xx Breve descrizione
+     V*=====================================================================
+     V* 24/09/24  V6R1   BMA Creato
+     V*=====================================================================
+     FJALOGT0F  UF A E             DISK
+     DJALOGT         E DS                  EXTNAME(JALOGT0F) INZ
+     DJALOGT2        E DS                  EXTNAME(JALOGT0F) INZ
```

Comment:
@dom-apuliasoft It’s not possible to define two data structures based on a file without renaming the fields. The fields become duplicated, and the program cannot be compiled on IBMi. You changed the example without testing that it works on IBMi.